src/pybind/mgr/dashboard/controllers/multi_cluster.py

```diff
@@ -320,13 +330,63 @@ def check_token_status(self, clustersTokenMap=None):
         clusters_token_map = json.loads(clustersTokenMap)
         return self.check_token_status_array(clusters_token_map)
 
+    @Endpoint()
+    @ReadPermission
+    def get_prometheus_api_url(self):
+        prometheus_url = Settings.PROMETHEUS_API_HOST
+        if prometheus_url is not None:
+            # check if is url is already in IP format
+            pattern = r'^(?:https?|http):\/\/(?:\d{1,3}\.){3}\d{1,3}:\d+$'
+            valid_ip_url = bool(re.match(pattern, prometheus_url))
```

Comment:
are we supporting IP addrs only in this case? please, can you justify why names are not allowed here?

Reply:
Yes, we are allowing just IP addresses here. The reason for that is prometheus is not able to resolve the fqdn of the remote cluster that is being added. So, each time the user has to add the host in /etc/hosts. To avoid this we are directly dealing with IP addrs here

Reply:
Please, keep in mind that some users may try ipv6 as well